core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -543,6 +543,16 @@ package object config {
       .version("1.2.0")
       .fallbackConf(DYN_ALLOCATION_SCHEDULER_BACKLOG_TIMEOUT)
 
+  private[spark] val LEGACY_LOCALITY_WAIT_RESET =
+    ConfigBuilder("spark.locality.wait.legacyResetOnTaskLaunch")
+    .doc("Whether to use the legacy behavior of locality wait, which resets the delay timer " +
+      "anytime a task is scheduled. See Delay Scheduling section of TaskSchedulerImpl's class " +
+      "documentation for more details.")
+    .internal()
+    .version("3.0.0")
```

Comment:
I think this was not merged into 3.0 branch, right?

Reply:
afaik that is correct we didn't backport it.
here is PR fixing these: https://github.com/apache/spark/pull/28307